lib/Pipelines/ArithmeticPipelineRegistration.h

```diff
@@ -66,6 +66,10 @@ struct MlirToRLWEPipelineOptions : public SimdVectorizerOptions {
       *this, "scaling-mod-bits",
       llvm::cl::desc("The number of bits in the scaling modulus for CKKS"),
       llvm::cl::init(45)};
+  PassOptions::Option<int> modBits{
+      *this, "mod-bits",
```

Comment:
```suggestion
      *this, "bfv-mod-bits",
```

If it's bfv-specific maybe signify that in the flag name